Header header
int8 mode
geometry_msgs/Vector3 angle
Armor target
int16 armor_count
Armor[] armor_list